composer.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "drupal/marketo_ma",
+  "type": "drupal-module",
+  "description": "Integration with Marketo marketing automation software.",
+  "keywords": ["Drupal"],
+  "license": "GPL-2.0+",
+  "homepage": "https://www.drupal.org/project/marketo_ma",
+  "minimum-stability": "dev",
+  "support": {
+    "issues": "https://www.drupal.org/project/issues/marketo_ma",
+    "source": "http://cgit.drupalcode.org/marketo_ma"
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/Jaesin/marketo-rest-api"
```

Comment:
Ah, my mistake. that is there until my pull request is accepted: https://github.com/dchesterton/marketo-rest-api/pull/34